src/main/java/com/jersey/representations/Cook.java

```diff
@@ -15,11 +15,11 @@
     @JoinColumn(name = "person_id")
     private Long person_id;
 
-    @OneToMany(fetch = FetchType.LAZY)
```

Comment:
I also did eager coz I wanted to write super fast and test. 
Lazy is better because then everytime the whole object is not loaded. Having json inside jsons and all of them being loaded would make it very slow. Lazy is the way to go.